keps/sig-node/3063-dynamic-resource-allocation/README.md

```diff
@@ -1332,8 +1335,9 @@ with “volume” replaced by “resource” and volume specific parts removed.
 This RPC is called by kubelet when a Pod that wants to use the
 specified resource is scheduled on a node.  The Plugin SHALL assume
 that this RPC will be executed on the node where the resource will be
-used.  The Plugin SHALL return device name and kind for allocated
-device[s].
+used.  Resource Claim UID and allocation result attributes should be
```

Comment:
On chat, we discussed that claim name/namespace/UID should be passed to identify the claim. They are useful for debugging and enable the resource driver to retrieve the full object, should that ever be needed (normally it shouldn't).

As it stands now, the KEP talks about a "Resource Claim UID" without defining what that UID is.

Reply:
Thanks for pointing out to this. Fixed.